tools/cldr-unittest/src/org/unicode/cldr/unittest/TestCLDRFile.java

```diff
@@ -844,4 +844,13 @@ public void TestExtraPaths() {
         assertTrue("Should be empty", missing.isEmpty());
     }
 
+    public void TestExtraPaths13954() {
+        CLDRFile es = cldrFactory.make("es", true);
+        CLDRFile es_US = cldrFactory.make("es_US", true);
+        if (!es_US.getRawExtraPaths().containsAll(es.getRawExtraPaths())) {
+            TreeSet<String> missing = new TreeSet<>(es.getRawExtraPaths());
```

Comment:
Sets.difference(es.getRawExtraPaths(), es_US.getRawExtraPaths()) perhaps?
It's use the ordering of the first set, so maybe not ordered the same.